src/collapse/collapse.ts

```diff
@@ -9,8 +12,35 @@ import {Directive, Input} from '@angular/core';
   host: {'[class.collapse]': 'true', '[class.show]': '!collapsed'}
 })
 export class NgbCollapse {
+  /**
+   * If `true`, collapse will be animated.
+   *
+   * Animation is triggered only when clicked on triggering element
+   * or via the `.toggle()` function
+   */
+  @Input() animation = false;
+
   /**
    * If `true`, will collapse the element or show it otherwise.
    */
   @Input('ngbCollapse') collapsed = false;
```

Comment:
I don't remember why we are _renaming_ this input here ?

Reply:
My guess is just to match the directive name `ngbCollapse` AND avoid using expressions like `ngbCollapse = true` internally in favour of more readable `collapsed = true`

Reply:
Ok, my point here was: I would be more comfortable with `@Input() collapsed = false`
So internaly still readable, externally **more** readable.

```
<div ngbCollapse [collapsed]="collapsed" ></div>
```

But I guess, here, I am falling in the trap of personal preference 🤔 Forget about it!

Reply:
That would be a breaking change, no? And not really related to this PR tbh :)